README.md

```diff
@@ -1,11 +1,11 @@
 # acloong  [![Build Status](https://travis-ci.org/ACLoong/acloong.svg?branch=master)](https://travis-ci.org/ACLoong/acloong)  
-An easy to use library in Modern C++
+An easy-to-use library in Modern C++
 
-### 介绍
+### Introduction
 
-1. `acloong` is an easy to use common components library which is written in modern standard C++.
+1. `acloong` is an easy-to-use common component library which is written in modern standard C++.
 
-2. All of the `acloong` codes based on C++11 right now，and it does not depends on any thirdparty library.
+2. All of the `acloong` codes are based on C++11 right now，and they do not depend on any third party's libraries.
```

Comment:
`any third party's libraries.` --> `any third party's library.`?

Reply:
It is used in negative and interrogative sentences instead of some to express "some", followed by countable nouns in the plural and uncountable nouns in the singular. 
So，——>any third party's libraries might be better.
In affirmative sentences, it means "any one" and needs to be followed by countable nouns.